uiowa/uiowa
# Intranet login route

Resolves https://github.com/uiowa/uiowa/issues/5911. 

# To test
- Build passes (all intranet tests pass)
```
ddev blt ds && ddev drush @default.local en sitenow_intranet -y && ddev drush @default.local cim -y
```
- https://default.uiowa.ddev.site/ should show Unauthorized title and message.
- You should see a Hawkid link with https://default.uiowa.ddev.site/saml/login?destination=/ as the URL.
- https://default.uiowa.ddev.site/about Hawkid link should show https://default.uiowa.ddev.site/saml/login?destination=/about as the URL
```
ddev drush @default.local uli admin/people/create
```
- Add a user with no other roles than authenticated with the username `only_auth`.
- Masquerade as users with other roles and click around to ensure that you have access to everything you would expect.
- Log out.
```
ddev drush @default.local uli --name only_auth
```
- You should be able to access the user view and edit tabs.
- Click a link in the menu
- You should see the Access Denied title and message
- You should also see a logout link.
